packages/react-server-redux/package.json

```diff
@@ -26,7 +26,7 @@
     "redux": "^3.6.0"
   },
   "peerDependencies": {
-    "react": "~0.14.2 || ^15.1.0"
+    "react": "~0.14.9 || ^15.3.0 || ^16.0.0"
```

Comment:
Why are the older version numbers changing here?

Lerna will only hoist if version strings match exactly.

Reply:
`prop-types` is only compatible with these versions. In previous versions, React will log console warnings if we use `prop-types` with previous versions https://www.npmjs.com/package/prop-types